nginx/domains/firmware.ffmuc.net.conf

```
@@ -37,9 +37,10 @@ map $firmwareVersion $targetDirectory {
     # OpenWrt 23.05
     "~^2023\.12\.2-next$"               $gluon_v2023_2_x;
     "~^2024\.([0-9]|10|11)\.\d+-next$"  $gluon_v2023_2_x;
+    "~^2025\.\d+\.$"                    $gluon_v2023_2_x;
```

I think it needs one more `\d+` with the `$` at the end
```suggestion
    "~^2025\.\d+\.\d+$"                 $gluon_v2023_2_x;
```

---

@DasSkelett thanks! I found another issue with a `11]` instead of `11` in the previous matching